config/default.py

```diff
@@ -36,6 +36,7 @@ class Default:
     INIT_VERTEX: bool = os.environ.get("INIT_VERTEX", "True").lower() in ("true", "1")
 
     GENMEDIA_BUCKET: str = os.environ.get("GENMEDIA_BUCKET")
+    PUBLIC_BUCKET: str = os.environ.get("PUBLIC_BUCKET", "False")
```

Comment:
Can you change this to 

`PUBLIC_BUCKET: bool = os.environ.get("PUBLIC_BUCKET", "False").lower() in ("false", "0")`

This ensures that `PUBLIC_BUCKET` is correctly interpreted as a boolean.

Reply:
Adjusted, thank you